app/models/label.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Resposible for storing configurations
+class Label < ApplicationRecord
+  belongs_to :account
+
+  enum scope: { general: 0, restricted: 1 }
```

Comment:
Where is the third scope?
According to slack message it should be 3 scopes: private, application, client.
In your case restricted is a private, but what is a general?

@mod Can you clarify required scopes?

Reply:
@maksim-litvinov #306 was updated by @mod recently. Two scopes left.

Reply:
scope is 'public' or 'private'

Reply:
@mod @maksim-litvinov unfortunately `public`, `private` is reserved words, so I replaced them with synonyms `general`, `restricted`